lib/internals/routeHandler.js

```diff
@@ -86,13 +91,15 @@ class RouteHandler {
             return this.parallelActionHandler.bind(this)(action);
         }
 
-        Logger.Instance.debug('Route handler action starting', action, this);
+        Logger.debug('Route handler action starting', action, this);
 
         //require handler to be called as per configuration type
         const handler = require(action.type);
 
         this.contexts[action.name] = {
             server  : this.server,
+            Promise : this.Promise,
+            Logger,
```

Comment:
I'm very happy about this too! This will make logging in methods so much simpler!